orchagent/mirrororch.cpp

```diff
@@ -678,6 +679,10 @@ bool MirrorOrch::getNeighborInfo(const string& name, MirrorEntry& session)
 
             return true;
         }
+        case Port::SYSTEM:
```

Comment:
Do we actually need this? Is knowing route existing enough?

Reply:
mirrororch makes use of getNeighborInfo() to resolve mirror dstIp into neighbor, especially when mirror dstIp is actually a neighbor. Therefore, the change is still required to allow resolving mirror dstIp to remote neighbor in multi-asic.